bodhi/tests/client/test___init__.py

```diff
@@ -324,6 +324,46 @@ def test_severity_flag(self, send_request):
         ]
         self.assertEqual(send_request.mock_calls, calls)
 
+    @mock.patch.dict(client_test_data.EXAMPLE_UPDATE_MUNCH, {'severity': 'urgent'})
+    @mock.patch.dict(os.environ, {'BODHI_URL': 'http://example.com/tests/'})
+    @mock.patch('bodhi.client.bindings.BodhiClient.csrf',
+                mock.MagicMock(return_value='a_csrf_token'))
+    @mock.patch('bodhi.client.bindings.BodhiClient.send_request',
+                return_value=client_test_data.EXAMPLE_UPDATE_MUNCH, autospec=True)
+    def test_severity_and_debug_flags(self, send_request):
```

Comment:
This test is really about testing the debug flag, so let's name it ```test_debug_flag``` for simplicity.

Reply:
Will adjust